.github/workflows/scripts/check-collector-module-version.sh

```diff
@@ -31,42 +33,20 @@ get_collector_version() {
 check_collector_versions_correct() {
    collector_module="$1"
    collector_mod_version="$2"
-   incorrect_version=0
-   mod_files=$(find . -type f -name "go.mod")
+   echo "Checking $collector_module is used with $collector_mod_version"
 
    # Loop through all the module files, checking the collector version
    for mod_file in $mod_files; do
-      if grep -q "$collector_module " "$mod_file"; then
-         mod_line=$(grep -m1 "$collector_module " "$mod_file")
-         version=$(echo "$mod_line" | cut -d" " -f2)
-
-         # To account for a module on its own 'require' line,
-         # the version field is shifted right by 1. Match
-         # with or without a trailing space at the end to account
-         # for the space at the end of some collector modules.
-         if [ "$version" == "$collector_module" ] || [ "$version " == "$collector_module" ]; then
-            version=$(echo "$mod_line" | cut -d" " -f3)
-         fi
-
-         if [ "$version" != "$collector_mod_version" ]; then
-            incorrect_version=$((incorrect_version+1))
-            echo "Incorrect version \"$version\" of \"$collector_module\" is included in \"$mod_file\". It should be version \"$collector_mod_version\"."
-         fi
-      fi
+      sed -i'' "s|$collector_module [^ ]*|$collector_module $collector_mod_version|g" $mod_file
    done
-
-   echo "There are $incorrect_version incorrect \"$collector_module\" version(s) in the module files."
-   if [ "$incorrect_version" -gt 0 ]; then
-      exit 1
-   fi
 }
 
 MAIN_MOD_FILE="./go.mod"
 
 # Note space at end of string. This is so it filters for the exact string
 # only and does not return string which contains this string as a substring.
-BETA_MODULE="go.opentelemetry.io/collector "
-BETA_MOD_VERSION=$(get_collector_version "$BETA_MODULE" "$MAIN_MOD_FILE")
+BETA_MODULE="go.opentelemetry.io/collector"
+BETA_MOD_VERSION=$(get_collector_version "$BETA_MODULE " "$MAIN_MOD_FILE")
```

Comment:
should the same change be applied to RC_MODULE below?

Reply:
done!